src/config/dynamic-theme-fixes.config

```diff
@@ -3248,6 +3248,16 @@ INVERT
 
 ================================
 
+themoviedb.org
+
+CSS
+.card {
+    background-color: #1d1f20 !important;
```

Comment:
background-color: ${#dfe1e2} !important;

Use ${} to darken that color and hold the original color in light mode.